spec/features/webauthn/submit_logging_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'rails_helper'
+
+RSpec.feature 'webauthn sign in' do
```

Comment:
Could we incorporate this into existing "webauthn setup" shared examples rather than adding a new file?

https://github.com/18F/identity-idp/blob/main/spec/support/shared_examples/webauthn_setup.rb

Reply:
You'll have to run the test files that reference the shared examples with `it_behaves_like`:

https://github.com/18F/identity-idp/blob/d5f82fb36ac9bf9ab5ec13889a2067851b857103/spec/features/webauthn/management_spec.rb#L9

https://github.com/18F/identity-idp/blob/d5f82fb36ac9bf9ab5ec13889a2067851b857103/spec/features/webauthn/sign_up_spec.rb#L27